CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -15,6 +15,17 @@ public class HarvestGateway : IHarvestDeveloperRetriever, ITimeSheetRetriever
         private readonly string _token;
         private readonly string _accountId;
         private readonly string _userAgent;
+        private readonly List<string> _listOfDeveloperRoles = new List<string>
```

Comment:
Could this be an environment variable?

Reply:
For this PR this is probably unnecessary - we will do this as a separate task.

Reply:
We'll be working on this next and putting it into a separate PR. We'll send you a message when it's ready.